### ISSUE TITLE
Latest Donation section on Donation page appears empty

### ISSUE THREAD

**Issue by BlinxCrunch02** [Alovoa/alovoa#186]
It doesn't load at all. Only the Largest Donation page has results appearing.

**Comment by Nonononoki** [comment]
There should be a single entry on the latest donation page, are you sure about this?

**Comment by BlinxCrunch02** [comment]
Yes, I'm sure. I tried it with several types of browsers both hardened and unhardened and the issue still persists.
